website/package.json

```diff
@@ -48,7 +48,7 @@
     "eslint-plugin-react": "7.11.1",
     "husky": "1.1.2",
     "lint-staged": "7.3.0",
-    "mockdate": "^2.0.2",
+    "mockdate": "2.0.2",
     "react-json-view": "^1.19.1",
```

Comment:
this should be removed - i dont think this is used on our website.

Reply:
mockdate is used in the tests to mock dates to have some consistency in eventcalendar tests

Reply:
its a dev dependency its only used in testing